crates/sui-json-rpc/src/api/governance.rs

```diff
@@ -15,6 +15,10 @@ use sui_types::sui_system_state::sui_system_state_summary::SuiSystemStateSummary
 #[open_rpc(namespace = "sui", tag = "Governance Read API")]
 #[rpc(server, client, namespace = "sui")]
 pub trait GovernanceReadApi {
+    /// Return one or more [DelegatedStake]
+    #[method(name = "getStakes")]
+    async fn get_stakes(&self, staked_sui_id: Vec<ObjectID>) -> RpcResult<Vec<DelegatedStake>>;
```

Comment:
```suggestion
    #[method(name = "getStakesByID")]
    async fn get_stakes_by_id(&self, staked_sui_id: Vec<ObjectID>) -> RpcResult<Vec<DelegatedStake>>;
```
nit: I would add this to distinguish from `getDelegatedStakes`

Reply:
since we have decided to not use the term delegation, lets rename the methods to `getStakes` and `getStakesByIds`